starter-code/css/main.css

```diff
@@ -6,3 +6,146 @@ Green: #00B172
 White: #FFF
 
 */
+
+@import url('https://fonts.googleapis.com/css?family=Montserrat:200,400,700');
+body{
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+}
+
+/*START NAV*/
+
+nav {
+    background-color: white;
+    height: 100px;
+}
+
+.spotify-icon{
+    height: 74px;
+    width: auto;
+    margin: 13px;
+}
+
+nav ul{
+    float: right;
+    margin: 0;
+    width: 37%;
+}
+
+nav li {
+    padding: 20px;
+    margin: 18px 0;
+    float: left;
+    list-style-type: none;
+}
+
+nav li a{
+    text-decoration: none;
+    color: black;
+    font-size: 1.2em;
+}
+/*START HEADER*/
+
+header{
+    background-image: url("../images/landing.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 650px;
+    clear: both;
+    color: white;
+    text-align: center;
+    font-size: 2.3em;
+    padding-top: 300px;
+}
+
+/*START WHITE SECTION*/
+
+.white-section{
+    background-color: white;
+    padding: 5%;
+
+}
+
+h2 {
+    text-align: center;
+    font-size: 2.2em;
+    font-weight: bold;
+}
+
+h2 hr{
+    border-style: solid;
+    border-width: 2px;
+    border-color: #00B172;
+    width: 25%;
+    margin: 8px auto;
+}
+
+.white-section div{
+    float: left;
+    height: auto;
+    text-align: center;
+    margin: 3% 0;
+}
+
+.each-icon-group{
+    float: left;
+    width: 30%;
+    padding: 1%;
+}
+
+.icons {
+    height: 140px;
+}
+
+.icon-title{
+    color: #00B172;
+    font-weight: bold;
+    font-size: 1.8em;
+}
+
+.under-icon-text{
+    font-size: 1.5em;
+    font-weight: 200;
+    padding: 30px;
+}
+
+/* START GREEN SECTION*/
+
+.green-section{
+    clear: both;
+    background-color: white;
+    color: white;
+    
+}
+
+.green-section-in{
```

Comment:
You should add the the logo spotify as a background image and center it.
This is the neat way.